hrjobcontract/js/test/controllers/form/form-leave_test.js

```diff
@@ -54,14 +56,27 @@ define([
           expect($scope.entity.leave[2].add_public_holidays).toBe(false);
         });
       });
+
+      describe('kek', function () {
+        beforeEach(function () {
+          $scope.$digest();
+          $scope.entity.leave[0].add_public_holidays = true;
+          $scope.entity.leave[0].leave_calculation_unit_name = 'hours';
+        });
+
+        it('sets the other leave types to not add public holidays', function () {
+          $scope.$digest();
```

Comment:
There is a call to digest cycle in the beforeEach, why do we need it here again? and cat we move it to beforeEach?